collectors/node/cpu.go

```diff
@@ -127,15 +129,19 @@ func init() {
 	lastCPU.Unlock()
 }
 
-func getCPUTimes() (cpu.TimesStat, cpu.TimesStat) {
-	currentTimes, _ := cpu.Times(false) // get totals, not per-cpu stats
+func getCPUTimes() (cpu.TimesStat, cpu.TimesStat, error) {
+	currentTimes, err := cpu.Times(false) // get totals, not per-cpu stats
+	if err != nil {
+		return cpu.TimesStat{}, cpu.TimesStat{}, err
```

Comment:
nit: consider it in future refactor
we should probably return a reference to `cpu.TimeStat{}` instead of actual structure. That way you can return `nil`s on error. Also, you are making hard copies of `cpu.TimeStat{}` here.